### PR TITLE
DDP-8603/OS-email-address-links (configs)

### PR DESCRIPTION
**Pull request by llobzhanidze**
add email links
https://broadinstitute.atlassian.net/browse/DDP-8603

### CODE REVIEW

**Review by gskhirel — COMMENTED** [review]
This configuration file is shared between OS and LMS, I think it is not a good idea to put os email address there.

File: pepper-apis/studybuilder-cli/studies/shared/family-history/snippets/family-history-section-intro.conf [hunk 1]
```diff
@@ -52,7 +52,7 @@
           {
             "language": "en",
             "text": """
-              Please do not hesitate to contact our team with questions regarding this survey."""
+              Please do not hesitate to contact our team at <a href="mailto:info@OSproject.org" class='maincolor'>info@OSproject.org</a> or 651-602-2020 with any questions about this survey."""
```
Comment (MishaK1204) [inline pepper-apis/studybuilder-cli/studies/shared/family-history/snippets/family-history-section-intro.conf]: Please add more descriptive class name:  e.g. emailColor

Comment (gskhirel) [reply pepper-apis/studybuilder-cli/studies/shared/family-history/snippets/family-history-section-intro.conf]: Please revert this change

**Review by gskhirel — COMMENTED** [review]
Revert the last change

**Review by LKapanad — APPROVED** [review]

**Pull request merged**